CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)*
 
 * The methods `setExcludedReferrers` and `getExcludedReferrers` have been added to the JavaScript tracker. They allow setting and receiving the referrers the JavaScript tracker should ignore. If a referrer matches an entry on that list, it will not be passed with the tracking requests and the attribution cookie will stay unchanged. This can for example be used if you need to forward your users to an external service like SSO or payment and don't want any visits or conversions being attributed to those services.
 
+### New config.ini.php settings
```

Comment:
This is no longer in 4.12.0 but 4.12.4 (if we merge it now).

Reply:
I didn't realise it was by patch version, but it makes sense hehe. I'll change this now.